packages/date-time/src/components/Calendar/examples/Calendar.Inline.Example.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { DefaultButton } from 'office-ui-fabric-react';
-import { addDays, getDateRangeArray } from 'office-ui-fabric-react/lib/utilities/dateMath/DateMath';
+import { addDays, getDateRangeArray } from '../../../utilities/dateMath/DateMath';
```

Comment:
@ecraig12345, @JasonGore - I think we're moving away from relative imports, right?

Reply:
I tried making this a direct import doing @uifabric/date-time/lib/utilities/... and got build failures when running 'npm start'. Any idea why that would be? Do I need to export the utilties and the DatePicker from the root of the @uifabric/date-time package and access just the package's root index, or is there maybe another reason the direct import would have failed?

Reply:
Maybe change this line in the project's webpack.serve.config.js:
`'@uifabric/date-time/lib': path.join(__dirname, 'lib'),`

to:
`'@uifabric/date-time/lib': path.join(__dirname, 'src'),`

Reply:
If you run `npm run just lint-imports` inside of `packages/date-time`, it will error. However, it doesn't error when you do a full monorepo build. It looks like the `lint-imports` task is not running as part of builds...

Reply:
![image](https://user-images.githubusercontent.com/26070760/58597350-6cf9e880-822c-11e9-808d-dd8936aa2647.png)

Reply:
It only runs when `--production` is provided. Which I'm not sure is desirable since it makes lint errors likely to go unnoticed until the PR build.

Reply:
Made #9275 to address this